lib/themes/dosomething/paraneue_dosomething/js/reportback/templates/reportback.tpl.html

```diff
@@ -1,10 +1,21 @@
 <% if (isListItem) { %><li><% } %>
-<div class="photo -stacked -framed" data-reportback-item-id="...">
+<div class="photo -stacked -framed" data-reportback-item-id="<%= id %>">
   <figure class="wrapper">
     <img src="data:image/gif;base64,R0lGODlhAQABAIAAAAUEBAAAACwAAAAAAQABAAACAkQBADs=" alt="<%= caption %>" data-src="<%= image %>" />
     <figcaption class="photo__copy">
       <p class="photo__caption"><%= caption %></p>
     </figcaption>
   </figure>
+
+  <% if (reactions.enabled) { %>
+  <ul class="form-actions -inline photo_actions">
+    <% for (i = 0, count = reactions.data.length; i < count; i++) { %>
```

Comment:
I think you still need a `var` or `let` here to avoid this being made into an implicit global, right? I'd also vote for using the `reactions.data.length` in the condition clause for readability, since it isn't a big performance hit to re-check the length property on each iteration.